@shared/constants/stellar.ts

```diff
@@ -1,9 +1,42 @@
+import StellarSdk from "stellar-sdk";
+
 export enum NETWORK_NAMES {
-  TESTNET = "Testnet",
+  TESTNET = "Test Net",
   PUBNET = "Main Net",
 }
 
 export enum NETWORKS {
   PUBLIC = "PUBLIC",
   TESTNET = "TESTNET",
 }
+
```

Comment:
Everything below was in `helpers/stellar`, but since these are static, I'm moving these into `constants`